libbeat/outputs/elasticsearch/config.go

```diff
@@ -34,16 +35,17 @@ const (
 
 var (
 	defaultConfig = elasticsearchConfig{
-		Protocol:    "",
-		Path:        "",
-		ProxyURL:    "",
-		Params:      nil,
-		Username:    "",
-		Password:    "",
-		Timeout:     90 * time.Second,
-		MaxRetries:  3,
-		TLS:         nil,
-		LoadBalance: true,
+		Protocol:         "",
+		Path:             "",
+		ProxyURL:         "",
+		Params:           nil,
+		Username:         "",
+		Password:         "",
+		Timeout:          90 * time.Second,
+		MaxRetries:       3,
+		CompressionLevel: 0,
```

Comment:
I somehow expected that some compression as default is good. I assume there is somehow a tradeoff between performance and compression level, but we can figure that out later.